src/averageAges.js

```diff
@@ -36,8 +45,22 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter(person =>
+      person.sex === 'f' && people.some(child => child.mother === person.name))
+    : people.filter(person => person.sex === FEMALE);
+
+  const womenTotalAge = women.reduce((acc, woman) => {
+    return acc + (woman.died - woman.born);
+  }, 0);
+
+  womenTotalAge.sort().reverse();
+
+  const averafeWoomanAge = womenTotalAge / women.length + 1;
```

Comment:
Looks like you have a typo in the variable name here. Also, you don't need to add 1 to the result, it will make the result inaccurate